packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -444,11 +532,65 @@ export const AddEditProviderModal: React.FunctionComponent<IAddEditProviderModal
                     />
                   </FormGroup>
                 ) : null}
-                {fields?.url ? (
+                {fields?.insecure? (
+                  <Checkbox
+                    label="Skip predefined certificate validation (if checked, a given certificate won't be validated)"
```

Comment:
```suggestion
                    label="Skip certificate validation (if checked, the certificate authority certificate will not be validated)"
```

this is an important distinction, current label may mislead users to understand that the systems CA will be validated, but if you check this field NO VALIDATION at all will take place.

IMHO the default should be `off`  so it will check the CA validation

https://docs.openstack.org/python-openstackclient/latest/cli/man/openstack.html#authentication-using-federation (look for the `--insecure` flag)
https://github.com/kubev2v/forklift/blob/de38ece340a72e0802ca5116be16afc9fce2c6b2/pkg/controller/provider/container/openstack/client.go#L52

cc:// @sgratch @ahadas

Reply:
@yaacov  @ahadas  Based on what we understood from @mmartinv and discussed on demo today, the systems CA will be validated in case of `insecure=true` (instead of the given CA), so there is a validation anyway. But maybe we understood it wrong,

I'm not sure what https://github.com/kubev2v/forklift/blob/de38ece340a72e0802ca5116be16afc9fce2c6b2/pkg/controller/provider/container/openstack/client.go#L78 is actually doing 

Based on this code https://youngkin.github.io/post/gohttpsclientserver/ :
"http.Transport contains a tls.Config. As with the servers, the Certificates field is populated with the client’s certificate. There is a new field here, the RootCAs field. The GoDoc describes this field as follows:

RootCAs defines the set of root certificate authorities that clients use when verifying server certificates. **If RootCAs is nil, TLS uses the host’s root CA set.**"

@mmartinv can you please help understanding?

Reply:
> RootCAs defines the set of root certificate authorities that clients use when verifying server certificates. If RootCAs is nil, TLS uses the host’s root CA set."

currect, but in our case we don't set rootCAs to nil , we set the `InsecureSkipVerify: true`

Reply:
Fixed to `insecure=false` as default and rephrasing the CA cert field label text.
We will change it back if needed after having a final decision on this issue.